python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -184,6 +184,53 @@ def inference(self,
 # on_fit_start (LightningModule method overwrite)
 def on_fit_start(self):
     self._ortsess_up_to_date = False
+    self.exit_onnx()
+
+
+def _forward_onnx(self, *args):
```

Comment:
What's the difference vs.  `inference`? Should you call `inference` (or have `inference` call  `_forward_onnx`)?

Reply:
I don't think I should add `batchsize` or other additional parameters here for the forward, that is kind of important for some users (e.g. Chronos users) I think and it's only available in `inference`.

Reply:
The should `inference` call `_forward_onnx`?

Reply:
emmm, but users may directly call `pl_model.inference(x, batch_size=32)` to control the memory usage.

Reply:
This `pl_model(x)` or `pl_model.forward(x)` API is only for those users who want to transform their code to onnxruntime backend with 1 line changes(change `pl_model.eval()` to `pl_model.eval_onnx()`).

Reply:
But may be they can control the batch size them selves by using a data loader like the one I write above. In that case, we may remove the `inference` method.
P.S. I agree `inference` and `forward` should not be provided to users at them same time with such similar functionalities

Reply:
Yes, the user can still call `inference`, but internally `inference` should call `_forward_onnx`, no?

Reply:
Oh, that's what you mean... I misunderstand that to be using `_forward_onnx` replace `inference`. Yes, I will change it. THX

Reply:
So `_forward_onnx` will always return numpy? Is it compactible with `forward`?

Reply:
And does conversion between numpy and tensor impact latency?

Reply:
not really, I did some tests and the impact is minor